source/quick-reference.txt

```diff
@@ -0,0 +1,468 @@
+
+     - .. io-code-block::
+          :copyable: true
+
+          .. input::
+             :language: java
+
+             coll.updateOne(
+                     Filters.eq("title", "Cosmos"),
+                     Updates.push("genres", "Educational"));
+                
+          .. output::
+             :language: json
+
+             { title: 'Cosmos', genres: [ 'Documentary', 'Educational' ], ...}
+
+   * - | **Replace a Document**
+       |
+       | `API Documentation <{+api+}/apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#replaceOne(org.bson.conversions.Bson,TDocument)>`__
+       | :ref:`Usage Example <java-usage-replaceone>`
+       | :ref:`Fundamentals <replace-operation>`
+
+     - .. io-code-block::
+          :copyable: true
+
+          .. input::
+             :language: java
+
+             coll.replaceOne(
+                     Filters.and(Filters.eq("name",  "Deli Llama"), Filters.eq("address", "2 Nassau St")),
+                     new Document("name", "Lord of the Wings").append("zipcode",  10001));
+
+          .. output::
+             :language: json
+
+             { name: 'Lord of the Wings', zipcode: 10001 }
+
+   * - | **Delete a Document**
+       |
+       | `API Documentation <{+api+}/apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#deleteOne(org.bson.conversions.Bson)>`__
+       | :ref:`Usage Example <java-usage-deleteone>`
+       | :ref:`Fundamentals <java-fundamentals-delete>`
+
+     - .. code-block:: java
+          :copyable: true
+
+          coll.deleteOne(Filters.eq("title",  "Congo"));
+
+   * - | **Delete Multiple Documents**
+       |
+       | `API Documentation <{+api+}/apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#deleteMany(org.bson.conversions.Bson)>`__ 
+       | :ref:`Usage Example <java-usage-deletemany>`
+       | :ref:`Fundamentals <java-fundamentals-delete>`
+
+     - .. code-block:: java
+          :copyable: true
+ 
+          coll.deleteMany(Filters.regex("title", "^Shark.*"));
+
+   * - | **Bulk Write**
+       |
+       | `API Documentation <{+api+}/apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#bulkWrite(java.util.List)>`__
+       | :ref:`Usage Example <java-usage-bulkwrite>`
+       | :ref:`Fundamentals <java-fundamentals-bulkwrite>`
+
+     - .. io-code-block::
+          :copyable: true
+
+          .. input::
+             :language: java
+
+             coll.bulkWrite(
+                     Arrays.asList(
+                            new InsertOneModel<Document>(
+                                    new Document().append("title", "A New Movie").append("year",  2022)),
+                            new DeleteManyModel<Document>(
+                                    Filters.lt("year",  1970))));
+
+          .. output::
+             :language: json
+
+             AcknowledgedBulkWriteResult {
+               insertedCount: 1,
+               removedCount: ...,
+               ...
+             }
```

Comment:
issue (non-blocking): Why include the output from this operation but not other operations like insert, update, and delete?

Reply:
Seemed like something that some people ask about, seems reasonable to remove.